apps/wpcom-block-editor/src/common/tracking/track-record-event.js

```diff
@@ -1,20 +1,33 @@
 /**
  * External dependencies
  */
-import { registerPlugin } from '@wordpress/plugins';
 import { isObjectLike, isUndefined, omit } from 'lodash';
 import debug from 'debug';
+
 const tracksDebug = debug( 'wpcom-block-editor:analytics:tracks' );
 
-// In case Tracks hasn't loaded
+// In case Tracks hasn't loaded.
 if ( typeof window !== 'undefined' ) {
 	window._tkq = window._tkq || [];
 }
 
 // Adapted from the analytics lib :(
 // Because this is happening outside of the Calypso app we can't reuse the same lib
 // This means we don't have any extra props like user
-const tracksRecordEvent = function( eventName, eventProperties ) {
+
+export default ( eventName, eventProperties ) => {
+	if (
+		undefined === window ||
+		undefined === window._currentSiteId ||
+		undefined === window._currentSiteType
+	) {
+		return;
```

Comment:
Noting that as a side-effect some Jetpack sites may not fire tracking events until they get a Jetpack plugin version that defines these values on the window object. 👍

Reply:
Ah chatting with @obenland it sounds like we'll be deferring Jetpack standalone site for now. Can we add a top check at the top of `common/tracking.js`, so we can return early? There's no need for us to listen to the redux events in that case.

```
// This file tracks Simple and Atomic Block-Editor events only.
if (
		undefined === window ||
		undefined === window._currentSiteId ||
		undefined === window._currentSiteType
	) {
		return;
}
```

Reply:
I've added a commit wich these changes and tested in `calypso`, `simple/wp-admin` and `jetpack/wp-admin`

`calypso` and `simple/wp-admin`
![image](https://user-images.githubusercontent.com/77539/61538306-7a9f4380-aa0f-11e9-96d2-cd330364ac21.png)

`jetpack/wp-admin`
![image](https://user-images.githubusercontent.com/77539/61538509-eaadc980-aa0f-11e9-8b2d-c6cc5a8d36ec.png)